platforms/nuttx/src/px4/common/cdc_acm_check.cpp

```diff
@@ -117,7 +119,17 @@ static void mavlink_usb_check(void *arg)
 		case UsbAutoStartState::disconnected:
 			if (vbus_present && vbus_present_prev) {
 				if (sercon_main(0, nullptr) == EXIT_SUCCESS) {
-					usb_auto_start_state = UsbAutoStartState::connecting;
+
```

Comment:
We could check for the existing of /dev/ttyACM0 (CDCACM_DEVNAME_FORMAT) here and not proceed.

Reply:
something like this?
```
	struct stat sb;
	if (::stat(USB_DEVICE_PATH, sb) == PX4_OK) {
		usb_auto_start_state = UsbAutoStartState::connected;
		break;
	}
```